apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/index.php

```diff
@@ -48,7 +60,7 @@ function enqueue_new_blocks_showcase_script() {
 	 * blocks are registered, so they're all available for filter that sets the
 	 * "New" category.
 	 */
-	enqueue_script( 'new-blocks-showcase', false );
+	enqueue_script( 'index', false );
```

Comment:
With the above two changes, this could be 
```php
enqueue_script( 'new-blocks-showcase', false );
enqueue_script( 'contextual-tips', false );
enqueue_style( 'contextual-tips' );